sdk/nodejs/policy/server.ts

```diff
@@ -90,7 +96,7 @@ async function getPluginInfoRpc(call: any, callback: any): Promise<void> {
 }
 
 // analyze is the RPC call that will analyze an individual resource, one at a time (i.e., check).
-function makeAnalyzeRpcFun(policyPackName: string, policyPackVersion: string, policies: Policy[]) {
+function makeAnalyzeRpcFun(policyPackName: string, policyPackVersion: string, policies: Policies) {
```

Comment:
How do we test if this is working as expected?

Reply:
I've manually tested it, but I will add some actual integration tests.